.github/CODEOWNERS

```diff
@@ -12,7 +12,22 @@
 # - You do not need to assign yourself as a codeowner (unless you are a project
 #   lead listed on the page above).
 
-
+# Blanket codeowners
```

Comment:
Add one newline above* (Each section is separated via 2 lines*)

Reply:
Done